<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品管理</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>901</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_store">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>901</width>
       <height>359</height>
      </rect>
     </property>
     <attribute name="label">
      <string>新品入库</string>
     </attribute>
     <widget class="QLabel" name="typeLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>类别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLalel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="purchasePriceLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>进价：</string>
      </property>
     </widget>
     <widget class="QLabel" name="amountLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddInStorage">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>入库</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearStorage">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清仓</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeSelect_store">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>选择商品类别</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="purchasePrice">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="salePriceLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>售价：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="salePrice">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="amount_add">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableView" name="StorageTable">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>531</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_buy">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>901</width>
       <height>359</height>
      </rect>
     </property>
     <attribute name="label">
      <string>预售订单</string>
     </attribute>
     <widget class="QLabel" name="amountLabel_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPayOrder">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>付款购买</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeSelect_buy">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>选择商品类别</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="amount_buy">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="singlePriceLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>单价：</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLalel_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="inventoryLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>库存：</string>
      </property>
     </widget>
     <widget class="QLabel" name="typeLabel_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>类别：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddInOrder">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加入订单</string>
      </property>
     </widget>
     <widget class="QComboBox" name="nameSelect">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>选择先商品类别</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="totalPriceLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>总价：</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_total">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_inventory">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_single">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="orderList">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>511</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
